### DISCUSSION TITLE
Certification of Tax Residence GitHub Inc.

### DISCUSSION THREAD

**Discussion by AdamDrozapol** [community/community#131530] [Other Feature Feedback, Questions, & Ideas]
Hi, 
I need Certification of Tax Residence GitHub Inc. for the accounting department.
Where can I find it?

**Comment by AdamDrozapol** [comment]
Just now, I received this document from GitHub Support by chat https://github.com/orgs/community/discussions

**Comment by ghostinhershell** [comment]
Hi @AdamDrozapol! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
